CSharp/UserManagement/User.cs

```diff
@@ -18,10 +18,7 @@ public User(int id)
 
         public int Id { get; }
 
-        public IReadOnlyCollection<int> Connections
-        {
-            get { return connections; }
-        }
+        public IReadOnlyCollection<int> Connections => connections;
```

Comment:
Just a note, if you should feel like using modern C# language features in other pull requests.

I've deliberately held this code base old-fashioned, since it's the code that supports two _Humane Code_ episodes, and those should preferably be understandable by other viewers than C# programmers on the cutting edge. Not only should that include C# programmers who are bit behind, but more importantly, Java programmers with an open mind.

I'm fairly sure that this class is never shown in the videos, so we can get away with a change like this here.

I'l decline any 'modernisation' of the core part of the code, though. I wish to keep it deliberately closer to a C-based lingua franca, and C# 7 and 8 tends to pull too much away from that.

Reply:
>I've deliberately held this code base old-fashioned [...]

Oh, fair enough. I wasn't fully aware of that.

>closer to a C-based lingua franca, and C# 7 and 8 tends to pull too much away from that.

Agreed on that. (I've actually stopped *constantly* following the latest and greatest on the C# side.) 

Perhaps, expression-bodied properties are 'prettier' than those getters, which aren't closer to C-based syntax either, I think...